client/network_create_test.go

```diff
@@ -53,10 +53,9 @@ func TestNetworkCreate(t *testing.T) {
 	}
 
 	networkResponse, err := client.NetworkCreate(context.Background(), "mynetwork", types.NetworkCreate{
```

Comment:
We need to update `client.NetworkCreate` to always send `CheckDuplicate: true` when creating an API request to an older API version.

Otherwise, older API's without this patch will use `CheckDuplicate: false` (on the daemon side), which would allow for duplicate networks to be created without error.